Panic when running on Docker

When running in Docker then a panic occurs when writing some certain data:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x123614b]

goroutine 270 [running]:
github.com/segmentio/parquet-go/encoding/rle.appendBitPackedInt32({0xc0008ac728, 0x1, 0x40d214}, {0xc004996000, 0x1b, 0x194}, 0x3)
	/Users/marty/src/tempo2/tempo/vendor/github.com/segmentio/parquet-go/encoding/rle/rle.go:478 +0xeb
github.com/segmentio/parquet-go/encoding/rle.encodeInt32({0xc0008ac728, 0x1, 0x8}, {0xc004996000, 0x1, 0x3}, 0x3)
	/Users/marty/src/tempo2/tempo/vendor/github.com/segmentio/parquet-go/encoding/rle/rle.go:222 +0x5e8
github.com/segmentio/parquet-go/encoding/rle.(*DictionaryEncoding).EncodeInt32(0x35e88e0, {0x0, 0xca5, 0x0}, {0xc004996000, 0x1278e02, 0x38})
	/Users/marty/src/tempo2/tempo/vendor/github.com/segmentio/parquet-go/encoding/rle/dictionary.go:30 +0x234
github.com/segmentio/parquet-go.indexedPageType.Encode({0x12a3e73}, {0x0, 0x0, 0x0}, {0xc004996000, 0xc0049bd500, 0x40d567}, {0x2455398, 0x35e88e0})
	/Users/marty/src/tempo2/tempo/vendor/github.com/segmentio/parquet-go/dictionary.go:1181 +0x3a
github.com/segmentio/parquet-go.(*writerBuffers).encode(0xc0009f4d80, {0x244b898, 0xc008431c70}, {0x2455398, 0x35e88e0})
	/Users/marty/src/tempo2/tempo/vendor/github.com/segmentio/parquet-go/writer.go:766 +0xa2
github.com/segmentio/parquet-go.(*writerColumn).writeBufferedPage(0xc000a6f1e0, {0x244b898, 0xc008431c70})
	/Users/marty/src/tempo2/tempo/vendor/github.com/segmentio/parquet-go/writer.go:1064 +0x125
github.com/segmentio/parquet-go.(*writerColumn).flush(0xc000a6f1e0)
	/Users/marty/src/tempo2/tempo/vendor/github.com/segmentio/parquet-go/writer.go:883 +0xaf
github.com/segmentio/parquet-go.(*writerColumn).writeRows(0xc000a6f1e0, {0xc0004f0300, 0x12, 0x20})
	/Users/marty/src/tempo2/tempo/vendor/github.com/segmentio/parquet-go/writer.go:953 +0x9e
github.com/segmentio/parquet-go.(*writer).WriteRows(0xc000a60300, {0xc00000d1d0, 0x1, 0x400})
	/Users/marty/src/tempo2/tempo/vendor/github.com/segmentio/parquet-go/writer.go:663 +0x145
github.com/segmentio/parquet-go.(*Writer).WriteRows(...)
	/Users/marty/src/tempo2/tempo/vendor/github.com/segmentio/parquet-go/writer.go:153
github.com/segmentio/parquet-go.(*Writer).Write(0xc0005af4c0, {0x1b0a320, 0xc0083cb260})
	/Users/marty/src/tempo2/tempo/vendor/github.com/segmentio/parquet-go/writer.go:141 +0x20e
```

I believe this is because a non-AVX2 fallback for `encodeInt32Bitpack` is not set here:  https://github.com/segmentio/parquet-go/blob/main/encoding/rle/rle_amd64.go#L20

Happy to provide more details about the hardware/software/container environment if needed.

--- a/encoding/rle/rle_amd64.go
+++ b/encoding/rle/rle_amd64.go
@@ -18,6 +18,7 @@ func init() {
 		encodeInt32Bitpack = encodeInt32BitpackAVX2
 	default:
 		encodeInt32IndexEqual8Contiguous = encodeInt32IndexEqual8ContiguousSSE
+		encodeInt32Bitpack = encodeInt32BitpackDefault
 	}
 }
 

